src/Workspaces/Core/MSBuild.BuildHost/Microsoft.CodeAnalysis.Workspaces.MSBuild.BuildHost.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
-    <TargetFrameworks>$(NetRoslynSourceBuild);net472</TargetFrameworks>
+    <TargetFrameworks>$(NetRoslynBuildHostNetCoreVersion);net472</TargetFrameworks>
```

Comment:
This means though that the .NET core build host will be a `net6.0` binary which pulls in all other Roslyn components at `netstandard2.0`. Want to make sure that is understood here.

Reply:
Yes, this is understood and fine since we only use a very tiny bit of those other assemblies. https://github.com/dotnet/roslyn/issues/70945 tracks removing the ProjectReferences entirely since we end up using only a tiny bit of code but end up deploying a lot of binaries in the process. I think severing the references too will also help with maintenance here simply because it means the cross-TFM adventures go away and this is a very isolated project.